README.md

```diff
@@ -1 +1,109 @@
-# angular-concepts-readme
+# Angular Core Concepts
+
+## Overview
+
+In this lesson, we are going to look at the core concepts that we will use in every Angular application. Some will be familiar to you, however some may seem a bit strange. Don't worry if they take time to sink in!
+
+## Objectives
+
+- Define MVC (Model-View-Controller), MVVM (Model-View-ViewModel)
+- Describe one-way and two-way data-binding
+- Describe Angular's two-way binding
+- Describe Dependency Injection
+- Describe a Single Page Application and common features
+- Demonstrate server-side rendering concepts
+
+## Single Page Applications
+
+Angular allows us to create single page applications (SPAs). The means that the user goes to one page and very rarely will have to refresh/leave that page to go to other parts of the website.
```

Comment:
Similar to the previous README, can you start with a few examples of SPAs that the reader might be familiar with? "What do web applications like x, y, and z have in common? Have you noticed that when you go to one page you very rarely will have to refresh/leave that page to go to other parts of the website? These are what we call single page applications." This will let the reader instantly make a connection to something they've seen before.